integration_tests/test_endpoints.py

```diff
@@ -234,3 +239,10 @@ def test_sync_streaming_model_endpoint(capsys):
                 )
         finally:
             delete_model_endpoint(create_endpoint_request["name"], user)
+
+
+@pytest.mark.skip(reason="test doesn't currently work, needs to figure out s3 fallback")
```

Comment:
for the integration tests before deploy, I'd like to check s3 as well. I had changed this skip to since the comment to a skipif to just skip for the circleci env